charm/trusty/calico-docker/hooks/hooks.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+
+import sys
+from path import path
+from charmhelpers.contrib.ansible import AnsibleHooks
+
+# Create the hooks helper, passing a list of hooks which will be
+# handled by default by running all sections of the playbook
+# tagged with the hook name.
+
+def hook_names(here=path(__file__).parent):
```

Comment:
It looks like the location of this comment screws with the linter, which leads it to think it's only got one blank line instead of two. I recommend moving it.